compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -1028,4 +1029,27 @@ std::optional<int> getGPUSubgroupSize(mlir::FunctionOpInterface func) {
   return std::nullopt;
 }
 
+void QueryMMAIntrinsics(mlir::ModuleOp moduleOp,
+                        SmallVector<IREE::GPU::MMAAttr> &mmaAttrs) {
+  IREE::GPU::TargetAttr target;
+
+  // Walk through all `func::FuncOp` operations in `moduleOp`.
```

Comment:
This comment doesn't clarify anything -- `moduleOp.walk` is a very basic function and the intention is obvious here.